internal/beater/beater.go

```diff
@@ -622,6 +630,10 @@ func (s *Runner) newFinalBatchProcessor(
 		v.OnInt(stats.ActiveBulkRequests)
 		v.OnKey("completed")
 		v.OnInt(stats.BulkRequests)
+		v.OnKey("active_created")
+		v.OnInt(stats.IndexersDestroyed)
+		v.OnKey("active_destroyed")
+		v.OnInt(stats.IndexersCreated)
```

Comment:
These names still read a bit awkwardly to me. Maybe rename to `output.elasticsearch.indexers.{created,destroyed}`?